help/destinations/catalog/personalization/pega-profile.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Pega Profile Connector
+description: Use the Pega Profile Connector for S3 in Adobe Experience Platform to export full or incremental, or both, profile data to Amazon S3 cloud storage. In Pega Customer Decision Hub, data jobs can be scheduled in Customer Profile Designer to import profile data periodically from Amazon S3 storage. 
+---
+
+# Pega Profile Connector
+
+## Overview {#overview}
+
+Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the Pega Customer Decision Hub profile. This connector helps setup initial export of profile data and also helps sync up new profiles periodically into Pega Customer Decision Hub.  Having up-to-date data in Customer Decision Hub provides a better and updated view of your customer base for next-best-action decisioning.
+
+>[!IMPORTANT]
+>
+>This documentation page was created by Pegasystems. For any inquiries or update requests, please contact Pega directly [here](mailto:support@pega.com).
+
+## Use cases 
+
+To help you better understand how and when you should use the [!DNL  Pega Profile Connector] destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case 1
+
+A marketer wants to initially setup Pega Customer Decision Hub with profile data loaded from Adobe Experience Platform. This is an initial full load followed by delta loads on a scheduled basis.
```

Comment:
minor, but I believe setup should be spelled as "set up" here, since it is not a noun.